Scripts/Spawner.gd

```diff
@@ -5,6 +5,9 @@ const NUM_ENEMIES = 5
 const POINT_VARIATION = 10.0
 const MAX_DELAY = 5.0
 
+# Signals
+signal enemy_spawned
```

Comment:
Are you going to be using this in the future? This signal isn't being used in the code.